### PR TITLE
Introduced an option to disable the WaylandGateway

### PR DESCRIPTION
**Pull request by DunderRoffe**
Since there is an "ENABLE" option for all other Gateways,  I added an option for the Wayland gateway as well.

@tobsan is there any reason why this was not done before or is it just a miss?

### CODE REVIEW

**Review by tobsan — CHANGES_REQUESTED** [review]
There are no options for FileGateway and EnvGateway. If we want to have options for all of them, those two should be included in that as well (and should default to ON also).

File: README.md [hunk 1]
```diff
@@ -53,6 +53,9 @@ To disable support for various gateways at compile time, set
 * -DENABLE_DEVICENODEGATEWAY=OFF (for device node gateway)
 * -DENABLE_DBUSGATEWAY=OFF (for dbus)
 * -DENABLE_CGROUPSGATEWAY=OFF (for cgroups)
+* -DENABLE_WAYLANDGATEWAY=OFF (for wayland)
+* -DENABLE_FILEGATEWAY=OFF (for mounting files)
+* -DENABLE_ENVGATEWAY=OFF (for manipulateing environment variables)
```
Comment (tobsan) [inline README.md]: manipulating*

**Review by tobsan — APPROVED** [review]
Just a typo, looks good apart from that.

File: README.md (hunk 1 above)
Comment (DunderRoffe) [reply README.md]: fixed

**Pull request merged**